spacecmd/src/spacecmd/system.py

```diff
@@ -4616,6 +4618,8 @@ def do_system_scheduleproductmigration(self, args):
     arg_parser.add_argument('-s', '--start-time')
     arg_parser.add_argument('-d', '--dry-run', action='store_true', default=False)
     arg_parser.add_argument('-c', '--child-channels')
+    arg_parser.add_argument('--allow-vendor-change', action='store_true', default=False)
```

Comment:
I am not 100% sure how the interaction between spacecmd and our java code works exactly. But could you try if using all the options except `allow-vendor-change` also schedules the SP migration successful?

Reply:
Thank you @parlt91 for the review. I checked it and it works as the default for the parameter is false.